src/pcs/c1/PageMods.js

```diff
@@ -0,0 +1,145 @@
+import BodySpacingTransform from '../../transform/BodySpacingTransform'
+import CollapseTable from '../../transform/CollapseTable'
+import DimImagesTransform from '../../transform/DimImagesTransform'
+import LazyLoadTransformer from '../../transform/LazyLoadTransformer'
+import PlatformTransform from '../../transform/PlatformTransform'
+import Scroller from './Scroller'
+import ThemeTransform from '../../transform/ThemeTransform'
+
+/**
+ * Executes common JS functionality the client should start, like hooking up events for
+ * lazy loading and table collapsing/expanding.
+ * Client side complement of server-side DOM transformations.
+ * Note: this will be called automatically when DOM is ready
+ * @param {!Window} window
+ * @param {!Document} document
+ * @return {void}
+ */
+const onPageLoad = (window, document) => {
+  const lazyLoader = new LazyLoadTransformer(window, 2)
+  lazyLoader.collectExistingPlaceholders(document.body)
+  lazyLoader.loadPlaceholders()
+
+  CollapseTable.setupEventHandling(window, document, true, Scroller.scrollWithDecorOffset)
+}
+
+/**
+ * @typedef {function} OnSuccess
+ * @return {void}
+ */
+
+/**
+ * Makes multiple page modifications based on client specific settings, which should be called
+ * during initial page load.
+ * @param {!Document} document
+ * @param {!{}} settings client settings
+ *   { platform, clientVersion, theme, dimImages, margins, areTablesCollapsed, scrollTop }
+ * @param {?OnSuccess} onSuccess callback
+ * @return {void}
+ */
+const setMulti = (document, settings, onSuccess) => {
```

Comment:
Do you plan on making use of the client version in this function later?  If not, it might be best to take it out of the example in the docs.

Reply:
> Do you plan on making use of the client version in this function later? 

Yes

> If not, it might be best to take it out of the example in the docs.

I want clients to get used to pass this info along from the start. See the `Guidelines` sections in the pcs.md file.